### PR TITLE
Add "Installing a cluster on IBM Z / LinuxONE in a restricted network"

### PR DESCRIPTION
**Pull request by rh-max**

### CODE REVIEW

File: installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+[IMPORTANT]
+====
+Additional considerations exist for non-bare metal platforms. Review the information in the
+link:https://access.redhat.com/articles/4207611[guidelines for deploying {product-title} on non-tested platforms] before you install an {product-title} cluster.
+====
+
+.Prerequisites
+
+* xref:../../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[Create a mirror registry on your bastion host] and obtain the `imageContentSources` data for your version of {product-title}.
+* Before you begin the disconnected install process, you must move or remove the existing `/usr/local/bin/openshift-install` bastion file. This ensures that the updated `/usr/local/bin/openshift-install` file is created during the disconnected install process.
```
Comment (ahardin-rh) [inline installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc:25]: s/install/installation

File: installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+Be sure to also review this site list if you are configuring a proxy.
+====
+
+include::modules/installation-about-restricted-network.adoc[leveloffset=+1]
+
+include::modules/installation-requirements-user-infra.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See https://www.ibm.com/support/knowledgecenter/en/SSB27U_7.1.0/com.ibm.zvm.v710.hcpa6/bhslzvs.htm[Bridging a HiperSockets LAN with a z/VM Virtual Switch] in the IBM Knowledge Center.
```
Comment (ahardin-rh) [inline installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc:53]: link: before each external link

File: installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+include::modules/installation-registry-storage-non-production.adoc[leveloffset=+3]
+
+include::modules/installation-complete-user-infra.adoc[leveloffset=+1]
+
+include::modules/installation-ibm-z-troubleshooting-and-debugging.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See also https://access.redhat.com/solutions/4387261[How to generate SOSREPORT within OpenShift4 nodes without SSH].
```
Comment (ahardin-rh) [inline installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc:98]: link:

Comment (ahardin-rh) [reply installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc:98]: s/OpenShift4/{product-title} version 4

File: modules/installation-requirements-user-infra.adoc [hunk 4]
```diff
@@ -161,9 +164,9 @@ On your z/VM instance, set up:
 [discrete]
 === Hardware requirements
 
-* 3 LPARs with 6 IFLs that support SMT2
+* 3 LPARs with 6 IFLs each that support SMT2
 * 1 or 2 OSA or RoCE network adapters, or both
-* Hipersockets, which are attached to a node either directly as a device or by bridging with one z/VM vSwitch to be transparent to the z/VM guest. To directly connect Hipersockets to a node you must set up a gateway to the external network via a RHEL 8 guest to bridge to the Hipersockets network.
+* Hipersockets, which are attached to a node either directly as a device or by bridging with one z/VM VSWITCH to be transparent to the z/VM guest. To directly connect Hipersockets to a node you must set up a gateway to the external network via a RHEL 8 guest to bridge to the Hipersockets network.
```
Comment (ahardin-rh) [inline modules/installation-requirements-user-infra.adoc:169]: node, you must

File: modules/installation-requirements-user-infra.adoc [hunk 5]
```diff
@@ -179,7 +182,7 @@ On your z/VM instances, set up:
 [discrete]
 === Disk storage for the z/VM guest virtual machines
 
-* FICON attached disk storage (DASDs). These can be z/VM minidisks, fullpack minidisks, or dedicated DASDs. To reach the minimum required DASD size for RHCOS installations, you need extended address volumes (EAV). If available, use HyperPAV to ensure optimal performance.
+* FICON attached disk storage (DASDs). These can be z/VM minidisks, fullpack minidisks, or dedicated DASDs. To reach the minimum required DASD size for RHCOS installations, you need extended address volumes (EAV). If available, use HyperPAV and High Performance FICON (zHPF) to ensure optimal performance.
```
Comment (ahardin-rh) [inline modules/installation-requirements-user-infra.adoc:185]: s/RHCOS/{op-system-first}  (see the modules/core-attributes.adoc file)

File: modules/ssh-agent-using.adoc [hunk 6]
```diff
@@ -100,6 +103,7 @@ $ ssh-keygen -t rsa -b 4096 -N '' \
     -f <path>/<file_name> <1>
 ----
 <1> Specify the path and file name, such as `~/.ssh/id_rsa`, of the SSH key.
+Make sure not to specify an existing SSH key, as it will be overwritten.
```
Comment (ahardin-rh) [inline modules/ssh-agent-using.adoc:106]: s/Make sure not to specify/Do not specify

**Comment by ahardin-rh** [comment]
@rh-max Overall, looks great! Just made some minor comments. Once those changes are complete and the commits are squashed down into one commit, we should be good to go! Did QE review yet? Thanks!

**Comment by rh-max** [comment]
@ahardin-rh Implemented all of your feedback and squashed. The QE gave the ACK. This is ready for merging. Thank you!

**Comment by ahardin-rh** [comment]
Thank you!

**Pull request merged**